crypto/aead/Cargo.toml

```diff
@@ -12,5 +12,7 @@ path = "src/lib.rs"
 
 [dependencies]
 anyhow = "1.0.65"
+rpassword = "7.2.0"
 ring = "0.16.20"
+hex = "0.4.3"
```

Comment:
instead of `hex` we should use `bitcoin_hashes::hex`  and the `ToHex` trait. currently, upstream rust-bitcoin is redoing their hashing anyway, so as soon as [this PR for the new hex crate lands](https://github.com/rust-bitcoin/rust-bitcoin/pull/1573) we can depend on that.